test/app/submissions_test.rb

```diff
@@ -106,6 +106,23 @@ def test_unlike_a_submission
     post "/submissions/#{submission.key}/unlike", {}, login(bob)
   end
 
+  def test_unlike_removes_unread_likes
+    fred = User.create({username: 'bob', github_id: 3, email: "bob@example.com"})
```

Comment:
Wait. That's exactly what you did. Sheesh. I should learn to read. Thank you :heart: